crates/polars-json/src/json/deserialize.rs

```diff
@@ -88,6 +88,17 @@ fn deserialize_list<'a, A: Borrow<BorrowedValue<'a>>>(
                 .try_push_usize(value.len())
                 .expect("List offset is too large :/");
         },
+        BorrowedValue::Static(StaticNode::Null) => {
+            validity.push(false);
+            offsets.extend_constant(1)
+        },
+        value @ (BorrowedValue::Static(_) | BorrowedValue::String(_)) => {
```

Comment:
It is a bit lenient with what regards a list in json, but I understand that this makes things easier.